### PR TITLE
travis: add sleep 1s to wait etcd startup.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@siddontang @tiancaiamao  PTAL

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ before_script:
   - cd $GOPATH/src/github.com/coreos/etcd
   - ./build
   - ./bin/etcd &
+  - sleep 5s
```
Comment (ngaut) [inline .travis.yml]: It's too long, 1 second should be enough.

Comment (qiuyesuifeng) [inline .travis.yml]: In my test, 5 seconds is long enough, but i am not sure 1 second is OK.

Comment (siddontang) [inline .travis.yml]: you can use a script to check whether etcd starts ok or not.

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**